.eslintrc.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
         ecmaFeatures: {
             jsx: true,
         },
+        project: ["./tsconfig.json"],
```

Comment:
This is required to have for `@typescript-eslint/strict-boolean-expressions` rule to work